doc/ug/install_instructions/index.md

```diff
@@ -148,6 +148,30 @@ $ make install
 
 After installation you need to add `/tools/verilator/$VERILATOR_VERSION/bin` to your `PATH` environment variable.
 
+## Verible
+
+Verible is an open source  SystemVerilog style linter and formatting tool. The style linter is relatively mature and we use it as part of our [RTL design flow]({{< relref "doc/ug/design" >}}). The formatter is still under active development, and hence its usage is more experimental in OpenTitan.
```

Comment:
nit here and below: one sentence per line.

Reply:
ah sure yes. thanks for the reminder.